pkg/cli/scan.go

```diff
@@ -629,89 +608,196 @@ func resolveInputFileFromArg(ctx context.Context, tmpdir, inputFilePath string)
 }
 
 // resolveInputForRemoteTarget takes the given input string, which is expected
-// to be the name of a Wolfi package (or subpackage), and it queries the Wolfi
-// APK repository to find the latest version of the package for each
-// architecture. It then downloads each APK and returns a slice of file paths to
-// the downloaded APKs.
-//
-// For example, given the input value "calico", this function will find the
-// latest version of the package (e.g. "calico-3.26.3-r3.apk") and download it
-// for each architecture.
-func resolveInputForRemoteTarget(ctx context.Context, input string) (downloadedAPKFilePaths []string, cleanup func() error, err error) {
+// to be the name of a  package (or subpackage), and it queries the indices
+// repository to find the latest version of the package for the given architecture.
```

Comment:
nit:

```suggestion
// to be the name of a  package (or subpackage), and it queries the repository
// indices to find the latest version of the package for the given architecture.
```

tell me if I'm misunderstanding!